source/azure_iot_hub_client.c

```diff
@@ -29,7 +30,7 @@
 #endif /* azureiothubSUBACK_WAIT_INTERVAL_MS */
 
 #ifndef azureiothubUSER_AGENT
-    #define azureiothubUSER_AGENT    "DeviceClientType=c%2F" AZ_SDK_VERSION_STRING "%28FreeRTOS%29"
+    #define azureiothubUSER_AGENT    "DeviceClientType=c%2F" azureiotVERSION_STRING "%28FreeRTOS%29"
```

Comment:
Should we add Embedded SDK version into this string?

Reply:
I *think* we can get away without it so we should see if we can to save some bandwidth.  @wduraes - would we do anything if we had embedded SDK version FreeRTOS was using?

The FreeRTOS layer we need but we should know the embedded SDK version with some level of accuracy based on FreeRTOS (we could obviosuly submodule update embedded SDK without a version bump of middleware, so we couldn't know exactly, but I think that's OK).

Reply:
Yea I really don't think it would be that hard to derive. And it saves the bandwidth. Whenever we do a release, if we name them right, we can always go back to them and check the embedded version if we really need it.